client/cmd/dexc/main.go

```diff
@@ -175,3 +178,36 @@ func mainCore() error {
 
 	return nil
 }
+
+// promptShutdown checks if there are active orders and asks confirmation to
+// shutdown if there are. The return value indicates if it is safe to stop Core
+// or if the user has confirmed they want to shutdown with active orders.
+func promptShutdown(clientCore *core.Core) bool {
+	err := clientCore.Logout()
+	if err == nil {
+		return true
+	}
+	if !errors.Is(err, core.ActiveOrdersLogoutErr) {
+		log.Errorf("unable to logout: %v", err)
+		return false
+	}
+
+	fmt.Print("You have active orders. Shutting down now may result in failed swaps and account penalization.\n" +
```

Comment:
Change fmt prints to log?

Reply:
I kinda like this just an unstamped stdout message to go with the stdin scanner.  I don't see a lot of value in this prompt being logged. The shutdown messages that follow are logged so that seems good to me.